conf/modules.config

```diff
@@ -295,6 +295,27 @@ process { // SUBWORKFLOW: GFF_STORE
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
         ]
     }
+
+    withName: '.*:GFF_STORE:EXTRACT_CDS' {
+        ext.args = '-x'
```

Comment:
Also here: `params.add_attrs_to_gffread_fastas`

Now that I think of it, `add_attrs_to_proteins_cds_fastas` is a better and more informative name. What do you think?

Reply:
I agree, change made